crates/bdk/tests/wallet.rs

```diff
@@ -1568,8 +1590,9 @@ fn test_bump_fee_reduce_change() {
         .insert_tx(tx, ConfirmationTime::Unconfirmed { last_seen: 0 })
         .unwrap();
 
+    let feerate = FeeRate::from_sat_per_kwu(625); // 2.5 sat/vb
```

Comment:
The math is not matching for me.
Am I doing something wrong?

`dbg!(psbt.clone().extract_tx().weight().to_wu() as f64 * 2.5);`

returns `1140.0`

Reply:
It looks like you're multiplying the tx weight by 2.5, which doesn't result in a feerate. To arrive at units of sat/kwu, you could do something like

`fee_amount * 1000 / tx.weight().to_wu()`

This will be more accurate if you `extract_tx` only after it has been signed.

Reply:
Thanks!